lib/api/core/models/security/role.js

```diff
@@ -45,11 +44,6 @@ Role.prototype.isActionAllowed = function roleIsActionAllowed (request, kuzzle)
     promises = [],
     path = [];
 
-  // explicit actions on internalIndex must be denied if no explicit restriction on internalIndex is set:
-  if (request.input.resource.index === kuzzle.internalEngine.index && !this.allowInternalIndex) {
```

Comment:
Not needed anymore here, as this check is implemented within `storageEngine` since #538 (see https://github.com/kuzzleio/kuzzle/pull/538/files#diff-8927ec75654a199501c9220ef47f3b5cR982 )